cypress/e2e/pass-creation.cy.js

```diff
@@ -1,19 +1,18 @@
 describe("DataPass", () => {
-  before(() => {
-    cy.login("user@yopmail.com", "user@yopmail.com");
-  });
-
   beforeEach(() => {
-    Cypress.Cookies.preserveOnce("_session_id");
+    cy.login("user@yopmail.com", "user@yopmail.com");
```

Comment:
does this mean that we should login before each test ? is there no way to do it once for all tests ?